### DISCUSSION TITLE
ch3-asm/ch3-08-goroutine-id

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#608] [General]
# ch3-asm/ch3-08-goroutine-id

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch3-asm/ch3-08-goroutine-id.html

**Comment by laokiea** [comment]
main.getg: relocation target runtime.convT2E not defined
你好👋，go run . 运行后报上面的错，怎么解决
